test/cql-pytest/test_keyspace.py

```diff
@@ -130,9 +137,13 @@ def test_alter_keyspace_missing_rf(cql, this_dc, scylla_only, has_tablets):
 
 # Test trying to ALTER a keyspace with invalid options.
 # Reproduces #7595.
-def test_alter_keyspace_nonexistent_dc(cql, this_dc):
+def test_alter_keyspace_nonexistent_dc(cql, this_dc, has_tablets):
     with new_test_keyspace(cql, "WITH REPLICATION = { 'class' : 'NetworkTopologyStrategy', '" + this_dc + "' : 1 }") as keyspace:
-        with pytest.raises(ConfigurationException):
+        if has_tablets:
+            exception_type = InvalidRequest
+        else:
+            exception_type = ConfigurationException
```

Comment:
nit: in Pythonish this would be `exception_type = InvalidRequest if has_tablets else ConfigurationException`.
But not critical at all :)

Reply:
OK, I wasn't aware, fixed, thanks.